packages/runtime/src/stratus.ts

```diff
@@ -1418,19 +1423,21 @@ Stratus.Internals.LoadImage = (obj: any) => {
 
             const srcOriginProtocol: any = srcOrigin.startsWith('//') ? window.location.protocol + srcOrigin : srcOrigin
 
-            // Set up actions for onLoad and onError (if size doesn't exist, revert to srcOrigin)
+            // Set up actions for to load the smallest image first.
             if (type === 'img') {
                 // Add Listeners (Only once per Element!)
                 el.on('load', () => {
                     el.addClass('loaded').removeClass('loading')
+                    jQuery(this).remove() // prevent memory leaks
                 })
                 el.on('error', () => {
-                    // TODO: Go down in sizes before reaching the origin
+                    // This is the smallest size, so there is nothing to fallback to. Let's hope a bigger size below fixes the image
                     el.attr('data-loading', dehydrate(false))
-                    el.attr('src', srcOriginProtocol)
+                    // el.attr('src', srcOriginProtocol)
```

Comment:
I'd annotate that this got moved into the prefetch logic below.

Reply:
I was actually curious why this was here, as this is set already when the image first loads on dom. I removed it from here because there was no point in adding a broken image in the first place.